docs/operating-eck/troubleshooting/common-problems.asciidoc

```diff
@@ -84,3 +84,75 @@ type: Opaque
 ----
 
 Failure to do so can cause data loss.
+
+[id="{p}-{page_id}-pod-updates"]
+== Pods are not replaced after configuration update
+
+When updating an existing Elasticsearch cluster configuration, the update can fail while being be applied.  This happens when the operator is unable to apply the changes required by replacing the pods of a given the Elasticsearch cluster.
+
+This can be identified by the `AGE` of the pod being older than the configuration update or the `STATUS` of the pods
+
+[source,sh]
+----
+kubectl get pods
+
+NAME                      READY   STATUS    RESTARTS   AGE
+quickstart-es-default-0   1/1     Running   0          15m
+quickstart-es-default-1   1/1     Running   0          15m
+quickstart-es-default-2   1/1     Running   0          15m
+----
+
+The causes include
+
+- Elasticsearch cluster is not healthy
+
+The health of the cluster may look like
```

Comment:
Maybe this line is not needed?

Reply:
Yes, it is redundant with the section header.  Do we need some sort of transition from bullet point to the command like we have for the K8s resource section?

Reply:
Not always. In this case it's quite intuitive the logical sequence from text to code.

Reply:
```suggestion
```